.gitignore

```diff
@@ -1,3 +1,7 @@
 node_modules/
 dist/
 .env
+docker-compose.yml
```

Comment:
why do you ignore these files/dirs?

https://github.com/targeek/slm-data-exporter/blob/4a147a1fa5df8c52873e7fb00e0801ae0530b497/.gitignore#L4-L7